content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md

```diff
@@ -26,10 +26,14 @@ see the [Creating a cluster with kubeadm](/docs/setup/production-environment/too
 * Full network connectivity between all machines in the cluster (public or private network is fine).
 * Unique hostname, MAC address, and product_uuid for every node. See [here](#verify-mac-address) for more details.
 * Certain ports are open on your machines. See [here](#check-required-ports) for more details.
-* Swap configuration. The default behavior of a kubelet was to fail to start if swap memory was detected on a node.
-  See [Swap memory management](/docs/concepts/architecture/nodes/#swap-memory) for more details.
-  * You **MUST** disable swap if the kubelet is not properly configured to use swap. For example, `sudo swapoff -a`
-    will disable swapping temporarily. To make this change persistent across reboots, make sure swap is disabled in
+* Swap configuration: The default behavior of a kubelet is to fail to start if swap memory is detected on a node.
```

Comment:
We're describing prerequisites here. Turning off swap used to be a prerequisite, but actually it no longer is. We should move the whole discussion later in the page.

Reply:
Yeah that makes sense.
WDYT about between `Check required ports` and `Installing a container runtime`?